EXTENSIONS.md

```diff
@@ -0,0 +1,179 @@
+# Extensions
+
+Podman Desktop is organized so that you can modularly add new functionality in the form of "extensions" as well as the corresponding `extension-api`. This allows you to communicate with Podman Desktop without having to know the internal-workings. You look for the API call and Podman Desktop will do the rest.
+
+Bundled extensions are located in the `/extensions` folder.
+
+#### Creating a new extension
+
+When creating a new extension, import the extension API: `import * as extensionApi from '@podman-desktop/api';` 
+
+All functionality with Podman Desktop is communicated through this API including registering the new extension. The API is located [here](https://github.com/containers/podman-desktop/blob/main/packages/extension-api/src/extension-api.d.ts).
```

Comment:
I think registering is ambiguous: what is registered ? The extension ? I think this is related to specific points and needs to be detailed